exchange/exchange.go

```diff
@@ -1315,6 +1315,14 @@ func makeBidExtJSON(ext json.RawMessage, prebid *openrtb_ext.ExtBidPrebid, impEx
 		prebid.Meta = &metaContainer.Prebid.Meta
 	}
 
+	if prebid.Meta == nil {
+		prebid.Meta = &openrtb_ext.ExtBidPrebidMeta{}
+	}
+
+	if prebid.Meta.AdapterCode == "" {
+		prebid.Meta.AdapterCode = adapter.String()
```

Comment:
The adapter code can always be set, without regard to what the bid response says (no need for the empty string check). Otherwise, it may not be correct.

Reply:
updated